File: Modul-6/Python/PRAK603-2310817320007-ZahraNabila.py
Baris1, Baris2 = map(int, input().split())

if(Baris1 != Baris2):
    print("Jumlah tidak sama")

else:
    Bil1 = list(map(int, input().split()))
    Bil2 = list(map(int, input().split()))

    for x in range (Baris1):
        print(Bil1[x] * Bil2[x], end=' ')

File: Modul-6/C/PRAK603-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int Baris1, Baris2, x, Isi, Hasil;
    scanf("%d %d", &Baris1, &Baris2);

    int bil1[Baris1], bil2[Baris1];
    if (Baris1 == Baris2)
    {
        for (x = 0; x < Baris1; x++)
        {
            scanf("%d", &Isi);
            bil1[x] = Isi;
        }
        for (x = 0; x < Baris1; x++)
        {
            scanf("%d", &Isi);
            bil2[x] = Isi;
        }
        for (x = 0; x < Baris1; x++)
        {
            Hasil = bil1[x] * bil2[x];
            printf("%d ", Hasil);
        }
    }
    else
    {
        printf("Jumlah tidak sama");
    }
    return 0;
}